specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json

```diff
@@ -435,15 +435,15 @@
               },
               {
                 "value": "OverQuota",
-                "description": "Reached daily cap quota, ingestion stopped."
+                "description": "Stopped ingestion following daily cap quota reach."
```

Comment:
keep it simple: "Daily quota has been reached and data ingestion was stopped."

Reply:
"Stop data ingestion following daily cap quota reach"

Reply:
I like mine better. I don't think we should use cap here. It's daily quota or daily cap. not both. And cap is kind of a word that has no dictionary entry. I prefer not to use it anyhow.